core/util/runTest.js

```diff
@@ -3,7 +3,7 @@ const Promise = require("bluebird");
 const fs = require("fs");
 const path = require("path");
 const chalk = require("chalk");
-const log = console.log;
+const { log } = console;
 
 const DEFAULT_METHOD = "get";
 let configs;
```

Comment:
I'm not a big fan of this, I reckon passing the config to each method is better rather than assigning to a global one

Reply:
Will update this as param instead of global var